docs/apis/subsystems/output/index.md

```diff
@@ -198,66 +198,76 @@ For each of the functions below we'll try to explain when they should be used, e
 
 ### String formatting functions
 
+The format_string and format_text functions should always be used when preparing the output of information. They may also be used to process information before it is stored in the database however, filters should only be applied at output. For example, language filters must only be applied as the content is prepared for output because we don't yet know the user's preferred language.
```

Comment:
```suggestion
The `format_string` and `format_text` functions should always be used when preparing the output of information. They may also be used to process information before it is stored in the database however, filters should only be applied at output. For example, language filters must only be applied as the content is prepared for output because we don't yet know the user's preferred language.
```

Generally best to put all method names in backticks.